### ISSUE TITLE
PS Move controller not pairing (PSMoveAPI update request)

### ISSUE THREAD

**Issue by GamerHun1238** [psmoveservice/PSMoveService#659]
I use Windows 10 1909 and PS Move controller is not pairing. Can you please update the PSMoveAPI version in the PSMoveService? (Sorry if I am bad at english)

**Comment by NikitaShokin** [comment]
Try #634

**Comment by RejectedHades** [comment]
I tried the admin build in #634 and it will not work for me. I am using a Model: CECH-ZCM2U. From the Skyrim bundle. Watched a video saying to use the admin build in the SteamVRBridge 1.5.1. That still didn't work.tried leaving it plugged in. Unplugging it as soon as I click the "Pair Controller" and unplugging it while I am pairing it. I have clicked the pop up and clicked allow to pair and I have also removed it before trying again and tried removing it repeatedly while I am pairing it. It will even ask me several time to allow it to pair. Just not sure what to try anymore...??
